app/main.py

```diff
@@ -1,3 +1,11 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args):
+        if args in cache_dict:
+            print("Getting from cache")
```

Comment:
When `args` are in cache, result should also be returned. This function returns result only in else branch

Reply:
oh, my bad, fixed

Reply:
Good job!